### PR TITLE
Do not target vue progressbar with server styles

### PR DESCRIPTION
**Pull request by marcoambrosini**
Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: core/css/inputs.scss [hunk 1]
```diff
@@ -885,7 +885,7 @@ div.select2-container {
 }
 
 /* Progressbar */
-progress {
+progress :not(.vue) {
```
Comment (juliushaertl) [inline core/css/inputs.scss:888]: Is the space intended?

Comment (ma12-co) [reply core/css/inputs.scss:888]: nope!

Comment (ma12-co) [reply core/css/inputs.scss]: pushed

**Review by juliusknorr — APPROVED** [review]

**Comment by skjnldsv** [comment]
I don't understand why.
Why would we need a vue component for this?

**Comment by ma12-co** [comment]
So? Are we getting this in?

**Comment by skjnldsv** [comment]
Sure, I'm not fond of this, but let get this in :)

**Review by skjnldsv — APPROVED** [review]

**Comment by ma12-co** [comment]
Erhm.. Do you know why tests are failing?

**Comment by nickvergessen** [comment]
Drone history is gone, try a rebase on master

**Comment by ma12-co** [comment]
done!

**Pull request merged**